include/extractor/traffic_lights.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef OSRM_EXTRACTOR_TRAFFIC_LIGHTS_DATA_HPP_
+#define OSRM_EXTRACTOR_TRAFFIC_LIGHTS_DATA_HPP_
+
+namespace osrm
+{
+namespace extractor
+{
+
+namespace TrafficLightClass
+{
+typedef std::uint8_t Direction;
```

Comment:
Nit. `using Direction = uint8_t;`

and why not make it enum btw?

Reply:
I've just copied an existing type that interfaces with lua. 
I didn't actually explore if enums work. I'll take a look.

Reply:
Yep, it appears to work :+1: